README.md

```diff
@@ -16,6 +16,18 @@ Dev instances should be using the `.env.development.local` file while production
 #### `USE_MOCKS` (boolean, default: `true`)
 a boolean used to determine whether the API calls will be against a real Entando Core or if they are just being mocked internally.
 
+#### `USE_MFE` (boolean, default: `false`)
+a boolean used to enable EPC's MFE rendering on AppBuilder.
```

Comment:
does this mean we can disable the feature and run AB without the need of any ECR or mocks, right?

Reply:
yes, that's the point